docs/UserGuide.md

```diff
@@ -61,9 +61,29 @@ Welcome HR executives and Department Heads! Coydir is a desktop app to efficient
 
 ## Using this Guide
 
+
 ### Navigating the User Guide
+This document might look daunting at first! But fear not, in this section, you will learn how to use 
+this user guide like a professional.
+
+####Interested in how to setup? 
+
+You can head over to [Quick Start](#quick-start) to install and run Coydir.
+
+####Interested in what Coydir can do?
+
+You can head over to [Features](#features) to learn how to use the features 
+offered in Coydir.
+
+Here are some symbols to look out for while using this guide:
+
+| Symbol               | Meaning                                      | 
+|----------------------|----------------------------------------------|
+| `code`               | Text relevant to commands or name of a file. |
+| :information source: | Information that is useful to take note of.  |
+| :bulb:               | Tips for Coydir Users.                       |
+| :exclamation:        | Be wary and proceed with caution.            |
 
-**[To be updated]**
```

Comment:
Additionally, can add something on the FAQ and Command Summary portion?

Just a simple one-liner to redirect users who are looking for the specific segment.